cogs/gatekeep/gatekeep.py

```diff
@@ -132,7 +136,36 @@ async def setThreshold(self, ctx: Context, threshold: int):
         else:
             await ctx.send("The value for the threshold should be greater than 0!")
 
-    @_gatekeep.command(name="days")
+    @settings.command(name="attachmentweight", aliases=["aw", "attachment", "attachments"])
+    @commands.guild_only()
+    @checks.mod_or_permissions(administrator=True)
+    async def setAttachmentWeight(self, ctx: Context, weight: int):
+        """Set the weight for how attachments in a message are scored.
+        Each attachment in a message will add this weight to the score when evaluating.
+
+        Parameters:
+        -----------
+        weight: int
+            Weight for which every attachment in a message will be scored.
+        """
+
+        if weight >= 0:
+            await self.config.guild(ctx.guild).get_attr(KEY_ATTACHMENT_WEIGHT).set(weight)
+            self.logger.info(
+                "%s#%s (%s) set the attachment weight to %s",
+                ctx.author.name,
+                ctx.author.discriminator,
+                ctx.author.id,
+                str(weight),
+            )
+            await ctx.send(
+                f":white_check_mark: **Gatekeep - Attachment Weight**: "
+                f"The attachment weight has been updated to **{weight}**"
```

Comment:
Nit: do we want an ending period?
```suggestion
                f"The attachment weight has been updated to **{weight}**."
```

Reply:
Applied this change to the attachment weight message, as well as any other messages that were missing the ending period.